content/zh-cn/docs/concepts/configuration/secret.md

```diff
@@ -645,7 +645,7 @@ Kubernetes 所提供的内置类型 `kubernetes.io/ssh-auth` 用来存放 SSH 
 <!--
 The SSH authentication Secret type is provided only for convenience.
 You can create an `Opaque` type for credentials used for SSH authentication.
-However, using the defined and public Secret type (`kubernetes.io/ssh-auth`) helps other
+However, using the defined and public Secret type (`kubernetes.io/tls`) helps other
```

Comment:
This fix should also be extended to [Line 655](https://github.com/kubernetes/website/blob/bbd2d2c4106c2e892b3561b763f8e8a72d2ad875/content/zh-cn/docs/concepts/configuration/secret.md?plain=1#L655) as well.

```diff
- 不过，使用预定义的、公开的 Secret 类型（`kubernetes.io/ssh-auth`）
+ 不过，使用预定义的、公开的 Secret 类型（`kubernetes.io/tls`）
```